proto/control-api/src/grpc/callback.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package medea_callback;
+
+service Callback {
+  rpc OnEvent (Request) returns (Response);
+}
+
+// 'on_join' callback in Control API spec.
+message OnJoin {}
+
+// 'on_leave' callback in Control API spec.
+message OnLeave {
+  // Reason of why `Member` leaves server.
```

Comment:
Use either single quotes `'` or backticks `` ` `` everywhere, but do not mix them in the same style for the same stuff. Also, try to follow the existing style (see `api.proto` file).